js/flickity.js

```diff
@@ -129,8 +130,16 @@ Flickity.prototype._create = function() {
   this.accel = 0;
   this.originSide = this.options.rightToLeft ? 'right' : 'left';
   // create viewport & slider
-  this.viewport = document.createElement('div');
-  this.viewport.className = 'flickity-viewport';
+  if (this.options.noDomMod) {
+    this.viewport = document.querySelector('.flickity-viewport');
```

Comment:
Hm... You are right: I didn't keep in mind, there might be several instances. I'll have this changed in awhile